modules/spo-configuring-webhooks.adoc

```diff
@@ -0,0 +1,45 @@
+// Module included in the following assemblies:
+//
+// * security/security_profiles_operator/spo-advanced.adoc
+
+:_content-type: PROCEDURE
+[id="spo-configuring-webhooks_{context}"]
+= Configuring webhooks
+
+Both profile binding and profile recording make use of webhooks. Their configuration (an instance of `MutatingWebhookConfiguration` CR) is managed by SPO itself and is not part of the deployed YAML manifests. While the defaults should be acceptable for the majority of users and the webhooks do nothing unless an instance of either `ProfileBinding` or `ProfileRecording` exists in a namespace, it might still be useful to configure the webhooks.
+
+To change the configuration of the webhook, the `spod` CR exposes a `webhookOptions` object that allows the `failurePolicy` and the `namespaceSelector` variables to be set. This allows you to set the webhooks to "soft-fail" or restrict them to a subset of a namespaces so that even if the webhooks failed, other namespaces or resources are not affected.
+
+.Procedure
+
+. Set the `binding.spo.io` webhook configuration to ignore errors and restrict it to a subset of namespaces labeled with `spo.x-k8s.io/enable-binding=true` by creating the following patch file:
```

Comment:
By default it is created now.

Reply:
@xiaojiey  do you mean the webhook is created by default now? Do we need to remove this procedure?